common/pulp/common/constants.py

```diff
@@ -69,13 +69,6 @@
 # Resource manager worker name
 RESOURCE_MANAGER_WORKER_NAME = 'resource_manager'
 
-# The amount of time (in seconds) between process wakeups to "heartbeat" and perform
-# their tasks.
-PULP_PROCESS_HEARTBEAT_INTERVAL = 5
-
-# The amount of time (in seconds) after which a Celery process is considered missing.
-PULP_PROCESS_TIMEOUT_INTERVAL = 25
```

Comment:
I had to remove these constants from common and add them to pulp.server.constants. For more info see https://pulp.plan.io/issues/3135#note-22.